clients/spark/core/src/main/scala/io/treeverse/clients/GarbageCollector.scala

```diff
@@ -5,7 +5,8 @@ import io.treeverse.clients.LakeFSContext.{
   LAKEFS_CONF_API_ACCESS_KEY_KEY,
   LAKEFS_CONF_API_SECRET_KEY_KEY,
   LAKEFS_CONF_API_URL_KEY,
-  LAKEFS_CONF_API_CONNECTION_TIMEOUT
+  LAKEFS_CONF_API_CONNECTION_TIMEOUT,
+  LAKEFS_CONF_API_READ_TIMEOUT
```

Comment:
```suggestion
  LAKEFS_CONF_API_CONNECTION_TIMEOUT_KEY,
  LAKEFS_CONF_API_READ_TIMEOUT_KEY
```

Reply:
opened https://github.com/treeverse/lakeFS/issues/3989 to follow up on these great comments